coherent_rates/plot.py

```diff
@@ -233,14 +290,30 @@ def plot_pair_system_evolution(
         state_scattered_evolved,
     ) = get_isf_pair_states(system, config, initial_state, times, direction)
 
-    fig, ax, _anim1 = animate_state_over_list_1d_x(state_evolved_scattered, ax=ax1)
-    fig, ax, _anim2 = animate_state_over_list_1d_x(state_scattered_evolved, ax=ax1)
+    fig, ax, _anim1 = animate_state_over_list_1d_x(
```

Comment:
Is this not generic (IE can be used in 2d and just show the state for a single x?